pkg/ironcore/get_volume_ids.go

```diff
@@ -18,12 +18,13 @@ import (
 	"context"
 
 	"github.com/gardener/machine-controller-manager/pkg/util/provider/driver"
+	"github.com/spf13/pflag"
 	"k8s.io/klog/v2"
 )
 
 // IronCoreCSIDriver is the CSI driver for ironcore provisioner
-const (
-	IroncoreCSIDriver = "ironcore-csi-driver"
+var (
```

Comment:
Please move this var to the `cmd/main.go` where the other flags are located.

Reply:
If I move this variable to main.go I will have to update ironcoreDriver structure and pass csi driver name via this struct.
So that GetVolumeIDs method can access it. Is that ok ?

Reply:
That is correct. But that is the cleaner approach by having flags at a central place and pass them on to where they are needed.

Reply:
Also can we call the `var` `CSIDriver` name. No need to prefix it.

Reply:
Done